5-flexbox/assets/styles.css

```diff
@@ -0,0 +1,57 @@
+*{
```

Comment:
* will be applied for all the elements.
```suggestion
body {
```